cfgov/cal/admin.py

```diff
@@ -51,28 +55,29 @@ def changelist_view(self, request, extra_context=None):
             if icsfile:
                 file_type = icsfile.content_type.split('/')[0]
 
-                if(file_type != "text"):
-                    error.append( (['There was an error with the File'],["Incorrect File Type"]) )
-                    context.update({ 'error': error,})
-                    return super(CFPBCalendarImportICSFileAdmin,self).changelist_view(request,context)
+                if (file_type != "text"):
+                    error.append((['There was an error with the File'],
+                                  ["Incorrect File Type"]))
+                    context.update({'error': error, })
+                    return super(CFPBCalendarImportICSFileAdmin,
+                                 self).changelist_view(request, context)
```

Comment:
This might be one of the stylistic personal preferences that you mention, but this feels hard to read versus the alternate

```py
return super(
    CFPBCalendarImportICSFileAdmin,
    self
).changelist_view(request, context)
```

Alternatively, because there are a few places in this function that use `super(CFPBCalendarImportICSFileAdmin, self).changelist_view`, could this code be cleaned up instead using something like:

```py
base_view = super(CFPBCalendarImportICSFileAdmin, self).changelist_view
...
base_view(request, context)
```